From af554c9a060f401a8177f63c9644a7bd82bb5c03 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:28 -0400
Subject: [fs] nfsd: get rid of unused macro definition

Message-id: <1403560314-32525-27-git-send-email-bfields@redhat.com>
Patchwork-id: 84459
O-Subject: [Patch RHEL7 026/172] nfsd: get rid of unused macro definition
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

Since defined in Linux-2.6.12-rc2, READTIME has not been used.

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: dfeecc829eb8e4ccbbab2ebc9b81b4cebec7fad4
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4xdr.c b/fs/nfsd/nfs4xdr.c
index b77f6bd..5bef9cb 100644
--- a/fs/nfsd/nfs4xdr.c
+++ b/fs/nfsd/nfs4xdr.c
@@ -103,11 +103,6 @@ xdr_error:     \
  (x) = (u64)ntohl(*p++) << 32;  \
  (x) |= ntohl(*p++);   \
 } while (0)
-#define READTIME(x)       do {   \
- p++;     \
- (x) = ntohl(*p++);   \
- p++;     \
-} while (0)
 #define READMEM(x,nbytes) do {   \
  x = (char *)p;    \
  p += XDR_QUADLEN(nbytes);  \
-- 
1.7.1